### PR TITLE
fix(jobs): use new docker image

### PR DESCRIPTION
**Pull request by bodinsamuel**
## Describe your changes

- Jobs Use new docker image 
Already deployed in staging, nb: requires manual change before deployment on prod

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -91,12 +91,15 @@ RUN true \
   && apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false
 
 # Do not use root to run the app
-USER node
+# BUT it does not work with secret mount (could not find a solution yet)
```
Comment (TBonnin) [inline Dockerfile:93]: when do we need secret mount?

**Review by TBonnin — COMMENTED** [review]
I don't remember why jobs is not yet using the unified image. Do you remember? @bodinsamuel

File: Dockerfile (hunk 1 above)
Comment (bodinsamuel) [reply Dockerfile:93]: To load Google credentials for bigquery

**Comment by bodinsamuel** [comment]
> I don't remember why jobs is not yet using the unified image.

No big reasons, it was just a matter of taking the time

File: Dockerfile (hunk 1 above)
Comment (TBonnin) [reply Dockerfile:93]: do we need to find a solution before merging this change?

Comment (bodinsamuel) [reply Dockerfile:93]: nope, just running as root is good, <i just left the comment to be sure the next person is not trying to put an user back

**Review by TBonnin — APPROVED** [review]

**Pull request merged**